xml/MAIN-SBP-SUMA-on-IBM-PowerVM.xml

```diff
@@ -6,452 +6,452 @@
 ]>
 
 <article role="sbp" xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
-  xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="art-sbp-amdepyc-sles12sp3"
```

Comment:
Please reset the whitespace changes in this file. It is completely reindented and basically unreviewable.